public/docs/_examples/structural-directives/dart/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: structural_directives
+description: Structural directives example
```

Comment:
Just to confirm, we decided to add sdk and dart_to_js_script_rewriter to all pubspecs.